plugins/woocommerce-admin/client/homescreen/activity-panel/orders/index.js

```diff
@@ -367,10 +367,4 @@ OrdersPanel.propTypes = {
 	orderStatuses: PropTypes.array,
 };
 
-OrdersPanel.defaultProps = {
-	orders: [],
-	isError: false,
-	isRequesting: false,
```

Comment:
I think it's a good opportunity to also remove these from propTypes from above 🤔